mycli/myclirc

```diff
@@ -11,6 +11,10 @@ smart_completion = True
 # lines. End of line (return) is considered as the end of the statement.
 multi_line = False
 
+# Destructive warning mode will alert you before executing a sql statement
+# that may cause harm to the database.
```

Comment:
Let's add examples for destructive commands. Like `drop database` or `shutdown`.